forge/src/main/java/whocraft/tardis_refined/common/data/LangProviderEnglish.java

```diff
@@ -50,7 +48,7 @@ protected void addTranslations() {
         add(BlockRegistry.FLIGHT_DETECTOR.get(), "Flight Detector");
 
         /*Items*/
-        add(ItemRegistry.PATTERN_MANIPULATOR.get(), "BasePattern Manipulator");
+        add(ItemRegistry.PATTERN_MANIPULATOR.get(), "Base Pattern Manipulator");
```

Comment:
I don't know how the original typo happened, but it should just be Pattern Manipulator

Reply:
Should be Pattern yeah

Reply:
it was introduced in the shell patterns commit lol, when BasePatterns was introduced